packages/client-sdk-nodejs/package.json

```diff
@@ -14,8 +14,9 @@
   },
   "scripts": {
     "prebuild": "eslint . --ext .ts",
-    "test": "jest --testPathIgnorePatterns auth-client.test.ts --maxWorkers 1",
+    "test": "jest --testPathIgnorePatterns=\"auth-client\\.test\\.ts|http-apis\\.test\\.ts\" --maxWorkers 1",
```

Comment:
is this the right syntax? I think [Jest CLI](https://jest-archive-august-2023.netlify.app/docs/27.x/cli#--testpathignorepatternsregexarray) lets you pass in an array